### PR TITLE
:wrench: add gistic

### PR DESCRIPTION
**Pull request by zhangb1**

### CODE REVIEW

File: gistic/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+RUN mkdir /home/gistic/MCRInstaller
+
+RUN cd /home/gistic/MCRInstaller && \
+   wget https://www.mathworks.com/supportfiles/downloads/R2014a/deployment_files/R2014a/installers/glnxa64/MCR_R2014a_glnxa64_installer.zip && \
+   unzip MCR_R2014a_glnxa64_installer.zip
+
+RUN cd /home/gistic && \
+    wget ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz && \
+    tar -zxvf GISTIC_2_0_23.tar.gz
```
Comment (migbro) [inline gistic/Dockerfile]: Maybe remove this package after you untar it? It adds 500MB to the image!

File: gistic/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
```
Comment (migbro) [inline gistic/Dockerfile]: Will this work with ubuntu 16 or 18? 14 doesn't have LTS anymore I don't think...

File: gistic/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+RUN mkdir /home/gistic
+WORKDIR /home/gistic
+
+RUN mkdir /home/gistic/MCRInstaller
+
+RUN cd /home/gistic/MCRInstaller && \
+   wget https://www.mathworks.com/supportfiles/downloads/R2014a/deployment_files/R2014a/installers/glnxa64/MCR_R2014a_glnxa64_installer.zip && \
+   unzip MCR_R2014a_glnxa64_installer.zip
+
```
Comment (migbro) [inline gistic/Dockerfile]: I'm not sure how, but also there are two copies of MCR installer zip, one here:
`/home/gistic/MCRInstaller` and here: `/home/gistic/MCR_Installer` the second one is weird.  I think both zipa should be removed after install, and that will shave another 1GB off the docker size.

Comment (zhangb1) [reply gistic/Dockerfile]: Thanks for checking ,will update

File: gistic/Dockerfile (hunk 2 above)
Comment (zhangb1) [reply gistic/Dockerfile]: Not working on 16 and 18

```
E: Unable to locate package libXext6
E: Unable to locate package libXt6
E: Unable to locate package libXmu6
E: Unable to locate package libXp6
```

Comment (migbro) [reply gistic/Dockerfile]: Yup, I see,t hanks!

**Pull request merged**